build_tools/fetch_sources.py

```diff
@@ -411,6 +413,12 @@ def main(argv):
         default=[("iree", ["third_party/flatcc", "third_party/benchmark"])],
         help="Specify which nested submodules to fetch (e.g., project1:nested_in_project1_1,nested_in_project1_2 project2:nested_in_project2)",
     )
+    parser.add_argument(
+        "--include-math-libraries",
+        default=True,
+        action=argparse.BooleanOptionalAction,
+        help="Include math libraries that are part of ROCM",
+    )
```

Comment:
IMO this should be sorted lower, maybe in the group of `--include-rocm-media` and `--include-iree-libs` (after compilers, rocm-systems, rocm-libraries, etc.)